lib/lightning/projects.ex

```diff
@@ -465,11 +465,11 @@ defmodule Lightning.Projects do
       end
 
     Repo.transaction(fn ->
-      jobs = project_jobs_query(project) |> Repo.all()
+      workflows = project_workflows_query(project) |> Repo.all()
 
-      jobs
-      |> Enum.each(fn job ->
-        Lightning.Jobs.update_job(job, %{"enabled" => false})
+      workflows
+      |> Enum.each(fn workflow ->
+        Lightning.Workflows.mark_for_deletion(workflow)
```

Comment:
with this PR, you've moved `:enabled` from Jobs to Edges. On this line you should go from:
```
Lightning.Jobs.update_job(job, %{"enabled" => false})
```
to:
```
Lightning.Triggers.update_trigger(trigger, %{"enabled" => false})
```
There may not be an `update_trigger` function and you may need to make one or call `Repo.update()`